R/score.R

```diff
@@ -35,37 +35,24 @@
 #' as well as the paper [Evaluating Forecasts with scoringutils in
 #' R](https://arxiv.org/abs/2205.07090).
 #'
-#' @param data A data.frame or data.table with the predictions and observations.
-#' For scoring using [score()], the following columns need to be present:
-#' \itemize{
-#'   \item `observed` - the observed values
-#'   \item `predicted` - predictions or predictive samples for one
-#'   observed value. (You only don't need to provide a prediction column if
-#'   you want to score quantile forecasts in a wide range format.)}
-#' For scoring integer and continuous forecasts a `sample_id` column is needed:
-#' \itemize{
-#'   \item `sample_id` - an index to identify the predictive samples in the
-#'   'predicted' column generated by one model for one observed value. Only
-#'   necessary for continuous and integer forecasts, not for
-#'   binary predictions.}
-#' For scoring predictions in a quantile-format forecast you should provide
-#' a column called `quantile`:
-#'   - `quantile`: quantile to which the prediction corresponds
-#'
-#' In addition a `model` column is suggested and if not present this will be
-#' flagged and added to the input data with all forecasts assigned as an
-#' "unspecified model").
-#'
-#' You can check the format of your data using [check_forecasts()] and there
-#' are examples for each format ([example_quantile], [example_continuous],
-#' [example_integer], and [example_binary]).
+#' @param data A data.frame or data.table with the following columns:
+#' - `observed` - observed values
+#' - `predicted` - predictions, predictive samples or predictive quantiles
+#' - `model` - name of the model or forecaster who made a prediction
+#'
+#' Depending on the forecast type, one of the following columns may be required:
+#' - `sample_id` - index for the predictive samples in the 'predicted' column
+#' - `quantile`: quantile-level of the corresponding value in `predicted`
+#'
+#' For more information see the vignettes and the example data
+#' ([example_quantile], [example_continuous],
+#' [example_integer], [example_point()], and [example_binary]).
 #'
 #' @param metrics the metrics you want to have in the output. If `NULL` (the
```

Comment:
Given how muich is now loaded into this arg this doesn't seem like enough documentation?

Reply:
There is more documentation for `score()` in #375. I'd suggest to resolve this comment for now and circle back to it when reviewing the change to documentation. Can also make an issue if you prefer.

Reply:
Opened an issue to doube-check again later (#393). Resolving for now.